crates/subgraph/src/multi_orderbook_client.rs

```diff
@@ -9,11 +10,29 @@ use crate::{
     OrderbookSubgraphClient, OrderbookSubgraphClientError, PaginationArgs,
 };
 
-#[derive(Debug, Clone, Deserialize)]
+#[cfg(target_family = "wasm")]
+use rain_orderbook_bindings::impl_wasm_traits;
+#[cfg(target_family = "wasm")]
+use serde_wasm_bindgen::{from_value, to_value};
+#[cfg(target_family = "wasm")]
+use wasm_bindgen::convert::{
+    js_value_vector_from_abi, js_value_vector_into_abi, FromWasmAbi, IntoWasmAbi,
+    LongRefFromWasmAbi, RefFromWasmAbi, TryFromJsValue, VectorFromWasmAbi, VectorIntoWasmAbi,
+};
+#[cfg(target_family = "wasm")]
+use wasm_bindgen::describe::{inform, WasmDescribe, WasmDescribeVector, VECTOR};
+#[cfg(target_family = "wasm")]
+use wasm_bindgen::{JsValue, UnwrapThrowExt};
+
+#[derive(Debug, Clone, Serialize, Deserialize, Tsify)]
+#[tsify(into_wasm_abi, from_wasm_abi)]
```

Comment:
do we really need these impls for `MultiSubgraphArgs`, I dont see it being used as a binding anywhere on the js-api/src/subgrapg/order.rs

Reply:
This one is used as a parameter in the `getOrders` function